startup/GafferScene/arnoldLights.py

```diff
@@ -64,6 +64,8 @@
 Gaffer.Metadata.registerValue( "ai:light:quad_light", "intensityParameter", "intensity" )
 Gaffer.Metadata.registerValue( "ai:light:quad_light", "exposureParameter", "exposure" )
 Gaffer.Metadata.registerValue( "ai:light:quad_light", "colorParameter", "color" )
+Gaffer.Metadata.registerValue( "ai:light:quad_light", "widthParameter", "height" )  # Width and height are reversed because of `visualiserOrientation`
```

Comment:
This seems fishy - perhaps a sign that we should change vertex ordering, or that `visualiserOrientation` was wrong?

Reply:
I suppose there's another possible approach, which is to leave the `visualiserOrientation` as it is with the 90 degree rotation, and also have `widthParameter = "width"` and `heightParameter = "height"`.

![image](https://user-images.githubusercontent.com/7873467/222257495-a03d97d0-194e-4441-8d49-0d70920eb2c5.png)

That scheme with a width of 4 and a height of 2 gives the light above.

To me, that has the x and y axes reversed and is what I've been trying to correct for. But from the point of view of the image, the axes are not reversed. Maybe people who use images on the quad light are accustomed to rotating it?

Reply:
The job of the visualiser is to match what Arnold will do, even if what Arnold does is a bit odd. The old `visualiserOrientation` accounted for the fact that Arnold would rotate the texture 90 degrees by default, and the new visualisation no longer matches Arnold's behaviour. Unfortunately, the new visualisation _does_ seem to match for width and height, so I think we do need something like the `uvOrientation` you suggested, to separate the texture orientation from the geometry orientation.